migration/migration-validators.md

```diff
@@ -49,19 +39,19 @@ To start the validator registration process, fund the validator address you just
 
 <Callout type='info'>
 
-You will need funds to pay for the validator registration transactions and for the deposit for your validator. If you don't have funds for these transactions, please use the **[Testnet Faucet](https://test.nimiq.watch/#faucet)** to request NIM for the test-run.
+You will need funds to pay for the validator registration transactions and for the deposit for your validator. If you don't have funds for these transactions, please fund your address!
 
 </Callout>
 
 ### Step 3: Run the validator registration tool
 
-For testing this in the PoW testnet, you must specify the arguments as follows:
+To register your validator and keys on the PoW Mainnet, you must specify the arguments as follows:
 
 ```shell
-node validator-registration.js --validator validator-keys.json --network test
+node validator-registration.js --validator validator-keys.json --network main
 ```
 
-The tool will connect to the Nimiq PoW testnet chain. Upon consensus, the tool creates and signs the validator registration transactions and sends them to the network, effectively registering your validator. Note that these transactions are a bundle of 6, and each one has a nominal fee of 1 Luna. Please ensure you have funded your validator address — as instructed in the previous step — as the transaction fees are paid from this address.
+The tool will connect to the Nimiq PoW Mainnet chain. Upon consensus, the tool creates and signs the validator registration transactions and sends them to the network, effectively registering your validator. Note that these transactions are a bundle of 6, and each one has a nominal fee of 1 Luna. Please ensure you have funded your validator address — as instructed in the previous step — as the transaction fees are paid from this address.
```

Comment:
Not sure if this is supported, @onmax might have a better idea on how to set long dashes
```suggestion
The tool will connect to the Nimiq PoW Mainnet chain. Upon consensus, the tool creates and signs the validator registration transactions and sends them to the network, effectively registering your validator. Note that these transactions are a bundle of 6, and each one has a nominal fee of 1 Luna. Please ensure you have funded your validator address -- as instructed in the previous step -- as the transaction fees are paid from this address.
```